cli/asc.js

```diff
@@ -21,6 +21,7 @@ const colorsUtil = require("./util/colors");
 const optionsUtil = require("./util/options");
 const mkdirp = require("./util/mkdirp");
 const EOL = process.platform === "win32" ? "\r\n" : "\n";
+const SEP = process.platform === "win32" ? path.win32.sep : path.sep;
```

Comment:
Still not 100% sure that this is going to work with a browser version of the `path` module. Would prefer hardcoding `? "\\" : "/"` to be sure, and to match what EOL does.